pkg/peertls/extensions/extensions_test.go

```diff
@@ -37,22 +37,23 @@ func TestHandlers_Register(t *testing.T) {
 		require.NoError(t, err)
 		chains = append(chains, identity.ToChains(chain))
 
+		index := i
```

Comment:
About this one I cannot say with a quick look, I'm not sure when the function passed to `extensions.NewHandleFactory` executes.

I suspect that it executed correctly because it's a test and it passed before and now with these changes.

Nonetheless it's saver to make these changes proposed by the linter.